Tests/TableView/TestTableViewModels.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 @testable import ReactiveLists
 
 struct TestCellViewModel: TableCellViewModel {
-    let rowHeight: CGFloat = 42
+    let rowHeight: CGFloat? = 42
```

Comment:
The bug was overriding a default variable but we forgot this `?` so it was a new variable.